tier1/checklist.md

```diff
@@ -0,0 +1,387 @@
+|--------------------------------|----------------------------------------------------|---------------------------------------------------------------------------------|
+| Code Reviewer's Recommendation | CODE REVIEWER 1  CODE REVIEWER 2   CODE REVIEWER 3 | [Approved/Needs Approval]  [Approved/Needs Approval]  [Approved/Needs Approval] |
+| ISSO                           | ISSO REVIEWER                                      | [Approved/Needs Approval]                                                       |
+| ISG Technical Approval         | ISG REVIEWER                                       | [Approved/Needs Approval]                                                       |
+| Business Owner(s)              | BUSINESS OWNER 1  BUSINESS OWNER 2                 | [Approved/Needs Approval]  [Approved/Needs Approval]                            |
+
+
+
+---
+
+
+
+### Flipping the Switch: Making the Repository Public
+
+Once the repository has passed outbound review, we are ready 
+
+*Repository Actions* 
+
+Please enable the following  features to enhane repository security and maintain code quality:
+
+- [ ] **Dependabot Alerts**
+A GitHub Feature. Get notified when one of your dependencies has a vulnerability
+
+- [ ] **Secret Scanning Alerts**
+A GitHub Feature. Get notified when a secret is pushed to this repository. Ideally set this up to run after each new commit is pushed to the repository
+
+- [ ] **Branch Protection**
+Ensures the Integrity of important branches by preventing unauthorized actions like force pushes and requiring pull request reviews with specific checks before merging. Dev and main should be protected branches in the repository.
+
+- [ ] **Git Branching**
+After making the repository public, make sure there is a coherent git branching pla in place. For example: agree to merge features related pull requests into dev but merge bug fixes into main instead of dev first.
+
+[^1]: theres a couple of links
+
+- [ ] **Add Repolinter GH Action to CI**
+For ongoing adherence to repository hygiene standards, integrate the [repolinter Github Action](https://github.com/DSACMS/metrics/blob/main/.github/workflows/checks.yml) into your CI pipeline. This addition enhances your workflow by automatically enforcing repository cleanliness standards.
+
+- [ ] **Optional:DCO (Developer Certificate of Origin)**
+Requires all commit messages to contain the <span style="color:green"><i>Signed-off-by</i></span> line with an email address that matches the commit author. The Developer Certificate of Origin (DCO) is a lightweight way for contributors to certify that they wrote or otherwise have the right to submit the code they are contributing to the project. The GitHub app to enforce DCO can be found [here](https://github.com/apps/dco) 
+
+_Communications & Rollout :mega:_
+
+
+Share the good news with communities both inside and outside CMS!
+
+- [ ] **Draft a launch announcement**
+
+Be sure to include the following information:
```

Comment:
Can you indent 353-361

Reply:
the indented lines are clear bullet points hope this is right

Reply:
Gotcha, if you can push your changes, I can take a look and see if it looks right